CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(naoqi_driver)
 
+add_definitions(-std=gnu++11)
```

Comment:
that enables extensions that are not necessarily C++11. Mostly, it should not matter. BWT, that's what I would call upstream :) : https://cmake.org/gitweb?p=cmake.git;a=blob;f=Modules/Compiler/GNU-CXX.cmake;h=e1c555b4bb485e58ef2e13770090221f10ea0ba9;hb=HEAD#l21 with usage in https://cmake.org/gitweb?p=cmake.git;a=blob;f=Modules/Compiler/GNU-CXX.cmake;h=e1c555b4bb485e58ef2e13770090221f10ea0ba9;hb=HEAD#l40
@alkino, any opinion ?
You also don't want an add_definitions there but a set(CMAKE_CXX_FLAGS ${CMAKE_CXX_FLAGS} whatever).